elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -603,7 +611,9 @@ def grade(element_html, data):
     if check_indentation:
         indentations = {ans["uuid"]: ans["indent"] for ans in true_answer_list}
         for ans in student_answer:
-            if ans["indent"] != indentations.get(ans["uuid"]):
+            if indentations.get(ans["uuid"]) != -1 and ans[
+                "indent"
+            ] != indentations.get(ans["uuid"]):
```

Comment:
Nit, maybe a little more readable?

```suggestion
            indentation = indentations.get(ans["uuid"])
            if indentation != -1 and ans["indent"] != indentation:
```

Reply:
sure! it looked a lot more readable before it got auto formatted.